src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -170,7 +171,10 @@ private void ProcessType(DatabaseItemType type, SmugglerResult result, BuildVers
             counts.Processed = true;
 
             if (counts is SmugglerProgressBase.CountsWithLastEtag countsWithEtag)
+            {
```

Comment:
Are you sure that this is indicative to both counters and attachments?
What if I only have one of them?